### PR TITLE
Updates to kiali and logging chart values.yaml

### PR DESCRIPTION
**Pull request by Ressetkk**
**Description**
Updated charts for logging and kiali based on suggestions from #12108
**Related issue(s)**

#12108

### CODE REVIEW

**Review by a-thaler — APPROVED** [review]

File: resources/kiali/values.yaml [hunk 1]
```diff
@@ -298,7 +306,8 @@ kiali:
     # a single, specific Kiali version, thus this setting may have no effect depending on how the
     # operator itself was configured.
     #    ---
-      image_name: "eu.gcr.io/kyma-project/tpi/kiali"
+    #.   ATTENTION: this setting is will not be effective as it got replaced by the global.images.kiali setting.
```
Comment (NHingerl) [inline resources/kiali/values.yaml:309]:
```suggestion
    #.   ATTENTION: This setting will not be effective. It has been replaced with the global.images.kiali setting. 
```

Are you sure this is the correct line for the Attention note? Why not put it right after line 300 so users can stop reading if they aren't curious?

File: resources/kiali/values.yaml [hunk 2]
```diff
@@ -323,7 +332,8 @@ kiali:
     # a single, specific Kiali version, thus this setting may have no effect depending on how the
     # operator itself was configured.
     #    ---
-      image_version: "1.38.1-e0da5a68"
+    #.   ATTENTION: this setting is will not be effective as it got replaced by the global.images.kiali setting.
```
Comment (NHingerl) [inline resources/kiali/values.yaml:335]:
```suggestion
    #.   ATTENTION: This setting will not be effective. It has been replaced with the global.images.kiali setting. 
```

Again, consider putting this right after line 321.

**Review by NHingerl — APPROVED** [review]

**Review by a-thaler — APPROVED** [review]

**Review by kwiatekus — APPROVED** [review]

**Pull request merged**